experiment_helpers/adjust_experimentSettings.m

```diff
@@ -0,0 +1,134 @@
+function [expt] = adjust_experimentSettings(expt, h_fig, adjustment)
+% This function can be called by pressing 'e' on the keyboard during an experiment ('e' for experiment settings) 
+% 
+% Currently only used by taimComp but theoretically could be used by any experiment that wanted to change any of these
+% parameters about an experiment: 
+% 
+% - LPC order
+% - Trial duration (particularly relevant for patient populations) 
+% - Target vowel durations (for duration training in compensation studies) 
+% 
+% inputs: 
+% 
+%   expt                    The expt structure for the experiment. You are almost certainly changing a setting that is saved
+%                           in expt. 
+% 
+%   h_fig                   The experiment figure handle so that you can display a pause message to the participant
+% 
+%   adjustment              The kind of adjustment you want to make. 
+% 
+% Supported adjustments: 
+% 
+%   'LPC' (or 'lpc')        The LPC order. This is resaved under expt.audapterParams.nLPC
+% 
+%   'trialdur'              The duration of the trial (stimulus presentation) 
+% 
+%   'targetdur'             The target duration of whatever word you are duration training on. Will give you the option to
+%                           change minimum or maximum. This will apply for ALL WORDS (not just one), with appropriate
+%                           adjustments made for aijWord
+% 
+% In all cases, the current value will be displayed to you, and then you will be asked to provide the new value. 
+%   
+% Initiated RPK 11/7/2022, based on adjustOSTs
+% 
+% 
+
+dbstop if error 
+
+%% Set exptOst so that you don't lose original expt 
+exptAdjust = expt; 
+
+% Save original expt as original 
+% TODO: the TRUE original should probably be save-saved? And then if you adjust multiple times you get different files... 
+save(fullfile(expt.dataPath, 'expt_orig.mat'), 'expt'); % Because in some cases you may have changed expt to have a different trackingFileName 
+fprintf('Original expt structure saved as expt_orig.mat. \n'); 
+
+
+%% Display faux pause information (without actually pausing) 
```

Comment:
I'm not sure why you say this isn't actually pausing. Unless you just mean that it's different than a participant-initiated pause, because the participant has to wait for the experimenter in this case.